eureka.go

```diff
@@ -105,18 +113,27 @@ func main() {
 		return
 	}
 
-	if (*encrypt == false && *decrypt == false) || *inFile == "" {
+	if len(flag.Args()) == 0 {
 		fmt.Println("===================ᕙ(⇀‸↼‶)ᕗ===================")
 		fmt.Println(" Eureka is a tool to help you encrypt/decrypt a file")
 		fmt.Println(" to encrypt:")
-		fmt.Println("     eureka -encrypt -file [your-file]")
+		fmt.Println("     eureka your-file")
 		fmt.Println(" to decrypt:")
-		fmt.Println("     eureka -decrypt -file [encrypted-file]")
+		fmt.Println("     eureka encrypted-file-end-with-dot-enc")
```

Comment:
that's not very clear to me : o why not just `your-file.encrypted`

Reply:
ha, you got me, should be `encrypted-file.enc` at that time